clvm-traits/src/lib.rs

```diff
@@ -90,9 +96,9 @@ mod tests {
     }
 
     #[test]
-    fn test_proper_list() {
+    fn test_list() {
```

Comment:
renaming `proper_list` -> `list` and `curried_args` -> `args` are not convtroversial. I would land that change right away.